iml-api/src/graphql.rs

```diff
@@ -651,7 +709,7 @@ impl MutationRoot {
         interval: GraphQLDuration,
         use_barrier: Option<bool>,
     ) -> juniper::FieldResult<bool> {
-        sqlx::query!(
+        let id = sqlx::query!(
```

Comment:
What is returned when the we hit the `ON CONFLICT DO NOTHING` part?

If it's the same id, do we end up scheduling the same timer again?

Reply:
I posted my results in the comment below.